lib/bunch/map.ex

```diff
@@ -27,7 +27,33 @@ defmodule Bunch.Map do
 
   """
   @spec map_values(%{k => v1}, (v1 -> v2)) :: %{k => v2} when k: any, v1: any, v2: any
-  def map_values(enum, f) do
-    enum |> Enum.into(Map.new(), fn {key, value} -> {key, f.(value)} end)
+  def map_values(map, f) do
+    map |> Enum.into(Map.new(), fn {key, value} -> {key, f.(value)} end)
+  end
+
+  @doc """
+  Moves value stored at `old_key` to `new_key`.
+
+  If `old_key` is not present in `map`, `new_key` is removed. If `new_key`
+  is present in `map`, it's value is overwritten.
+
+  ## Examples
+
+      iex> #{inspect(__MODULE__)}.move(%{a: 1, b: 2}, :a, :c)
+      %{b: 2, c: 1}
+      iex> #{inspect(__MODULE__)}.move(%{a: 1, b: 2}, :a, :b)
+      %{b: 1}
+      iex> #{inspect(__MODULE__)}.move(%{a: 1, b: 2}, :c, :b)
+      %{a: 1}
```

Comment:
Do you have a use case for this (I mean behaviour when old_key is not present)? I don't find it intuitive, wouldn't an error be better?